pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -2604,12 +2631,28 @@ public int getDatabaseMinorVersion() throws SQLException {
     return connection.getServerMinorVersion();
   }
 
-  public int getJDBCMinorVersion() throws SQLException {
-    return 0; // This class implements JDBC 3.0
+  /**
+   * Retrieves the major JDBC version number for this driver.
+   *
+   * @return JDBC version major number
+   */
+  @Override
```

Comment:
Any clue why do you copy javadocs for `@Override` methods?

AFAIK, javadoc build does handle that just fine.
See http://static.javadoc.io/org.postgresql/postgresql/9.4.1212.jre7/org/postgresql/Driver.html#getParentLogger() as an example.

So I would remove all the inherited copy-pasted javadocs unless they have some postgresql specifics.
For instance, `connect` method does have some PG specific, so it is LGTM. On the other hand, things like `Retrieves the major JDBC version number for this driver` add little value and can be inherited.

However, in this particular method adding some pgjdbc-specific example would help a lot. For instance "The method returns the driver major version, not the versions of the backend. This method would return something like 42 ad pgjdbc versions are like 42.x.y."